### ISSUE TITLE
Tensorboard: PyTorch writer.add_scalars() does not work as intented.

### ISSUE THREAD

**Issue by ariG23498** [wandb/client#1562]
I tried the code as follows:
```python
from torch.utils.tensorboard import SummaryWriter
import wandb
wandb.tensorboard.patch(tensorboardX=False, pytorch=True)
wandb.init()
writer = SummaryWriter()
scalars = {
    'scalar_1': 1.0,
    'scalar_2': 2.0,
    'scalar_3': 3.0
}
writer.add_scalars('test', scalars, global_step=1)
writer.close()
wandb.finish()
```

My observations:
1. There is a warning that I face with my implementation which is as follows:
```
wandb: WARNING When using several event log directories, please call wandb.tensorboard.patch(root_logdir="...") before wandb.init
```
I have tried figuring out the `root_dir` parameter, due to the unavailability of a docstring or documentation, I was not able to remove the warning.
2. If we ignore the warning the tensorboard integration in wandb does not really work like the one in colab.
|The file structure of the event files that are logged by the `SummaryWriter` are|But in the wandb folder the event files are not structured the same way|
|---|---|
|![image](https://user-images.githubusercontent.com/36856589/100377748-d9c78c00-3037-11eb-9f97-959f83fca1cd.png)|![image](https://user-images.githubusercontent.com/36856589/100377787-f663c400-3037-11eb-9040-e58ba85a4305.png)|

Here we do not see the folders namely `test_scalar_1`, `test_scalar_2`, `test_scalar_3`.

This leads us to the bug in the tensorboard integration

| **Colab tensorboard** | **Tensorboard in wandb**|
|---|---|
|![image](https://user-images.githubusercontent.com/36856589/100377922-42166d80-3038-11eb-998e-1e2eb599182f.png)|![image](https://user-images.githubusercontent.com/36856589/100378164-9588bb80-3038-11eb-885a-2aaaf4219cd5.png)|

Here we do not notice the anything in the `runs` section of `wandb tensorboard`.

I am suspecting a probable issue with https://github.com/wandb/client/blob/30f77c8a320fe42a519f76eb012e61101afad9ba/wandb/integration/tensorboard/monkeypatch.py#L90

**Comment by vanpelt** [comment]
You just need to call wandb.tensorboard.patch(root_logdir=x) the root logdir is the directory that tensorboard is using as its logdir.  I'm not sure what the default is, maybe os.getcwd but you can also specify a logdir directly in the SummaryWriter instance.

**Comment by ariG23498** [comment]
I tried with this code snippet:
```python
from torch.utils.tensorboard import SummaryWriter
import wandb
wandb.tensorboard.patch(root_logdir='runs')
wandb.init(entity="ariG23498", project="errors")
writer = SummaryWriter()
scalars = {
    'scalar_1': 1.0,
    'scalar_2': 2.0,
    'scalar_3': 3.0
}
writer.add_scalars('test', scalars, global_step=1)
writer.close()
wandb.finish()
```
The file structure the SummaryWriter creates is as follows:
![image](https://user-images.githubusercontent.com/36856589/100759328-7f9f4000-3416-11eb-902d-1c58a3a70b98.png)

Here I did mention the root_logdir but the warning still prevails.

**Comment by ariG23498** [comment]
Hey @vanpelt 
I tried doing this
```
from torch.utils.tensorboard import SummaryWriter
import wandb
wandb.tensorboard.patch(root_logdir='exp')
wandb.init(entity="ariG23498", project="errors")
writer = SummaryWriter("exp")
scalars = {
    'scalar_1': 1.0,
    'scalar_2': 2.0,
    'scalar_3': 3.0
}
writer.add_scalars('test', scalars, global_step=1)
writer.close()
wandb.finish()
```

And this does not seem to supress the warnings anyway but the wandb is able to get the event files and the folder structure.
![image](https://user-images.githubusercontent.com/36856589/100761042-6f886000-3418-11eb-80a8-7e3298df2d83.png)
![image](https://user-images.githubusercontent.com/36856589/100760956-57b0dc00-3418-11eb-8d00-c35569a3730d.png)

The tensorboard issue is resolved only when we specify the root_logidr in the patch and the SummaryWriter instance. The warning somehow are still showing.

If you wish to close the ticket I would not have a problem :smile:

**Comment by raubitsj** [comment]
This fix will go out in the 0.10.13 release
